### PR TITLE
feat: add qualified plugin

### PR DESCRIPTION
**Pull request by elevatebart**
Add qualified plugin for sales

### CODE REVIEW

File: plugins/qualified.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = async function fullStory({ context, options }) {
```
Comment (jaffrepaul) [inline plugins/qualified.js:1]: assuming this block was inspired by the fullstory implementation, but the function should get it's own name 😄

**Comment by jaffrepaul** [comment]
@elevatebart all of the additional plugins add non-insignificant time to page load and additional resources to fetch. What exactly does this plugin do? I'm sure there's value to the marketing team but I'd like for the PR description to indicate purpose and value so we have context 🙏

File: plugins/qualified.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const isProd = process.env.NODE_ENV === 'production'
+  return {
+    name: 'docusaurus-plugin-qualified',
+    injectHtmlTags({ content }) {
+      if (!isProd) {
+        return {}
```
[the remark this answers is not in the document]
Comment (elevatebart) [reply plugins/qualified.js:7]: it is only a short circuit to avoid pollution

**Comment by elevatebart** [comment]
> @elevatebart all of the additional plugins add non-insignificant time to page load and additional resources to fetch. What exactly does this plugin do? I'm sure there's value to the sales & marketing teams but I'd like for the PR description to indicate purpose and value so we have context 🙏

Unfortunately I have no idea I will let Ben Z and Kelly C explain. I am the messenger.

Sorry for the hustle.

**Pull request closed without merging**